packages/radix-ui-themes/src/components/container.tsx

```diff
@@ -24,7 +24,7 @@ const Container = React.forwardRef<ContainerElement, ContainerProps>(
     );
 
     const { className: innerClassName, style: innerStyle } = extractProps(
-      { width, minWidth, maxWidth },
+      { width, minWidth, maxWidth, height, minHeight, maxHeight },
```

Comment:
Felt weird that width style props go on the inner element, but not height. If container root gets a height, the inner element won't stretch vertically to fill it.